components/eamxx/src/share/field/field_tracking.hpp

```diff
@@ -45,7 +45,7 @@ class FieldTracking : public FamilyTracking<FieldTracking> {
   const atm_proc_set_type& get_customers () const { return m_customers; }
 
   // List of field groups that this field belongs to
-  const ekat::WeakPtrSet<const FieldGroupInfo>& get_groups_info () const { return m_groups; }
+  const ekat::WeakPtrSet<const FieldGroupInfo>& get_group_info () const { return m_groups; }
```

Comment:
This method could have kept the original signature, since in this case we do return all of the groups. My comment was only for the FM method, which returns the info only for one group. Not important though.

Reply:
Ahh sorry, just did a massive replace with sed.. Forgot this method existed, I'll change it back.